### ISSUE TITLE
Getting started guide fails due to invalid url to download JDK

### ISSUE THREAD

**Issue by JakobMaaloe** [atricore/atricore-josso-playground#2]
Hey,

Just tried out the Getting started guide (http://www.josso.org/getting-started.html).

It fails while running the "vagrant up --provider virtualbox" command. Output:
==> default: Running hooks in /etc/ca-certificates/update.d....done.
==> default: The command '/bin/sh -c DEBIAN_FRONTEND=noninteractive   apt-get update &&   apt-get install -y wget &&   wget -q --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u73-b02/jdk-8u73-linux-x64.tar.gz &&   mkdir -p /opt/jdk &&   tar -zxf jdk-8u73-linux-x64.tar.gz -C /opt/jdk &&   update-alternatives --install /usr/bin/java java /opt/jdk/jdk1.8.0_73/bin/java 100 &&   update-alternatives --install /usr/bin/javac javac /opt/jdk/jdk1.8.0_73/bin/javac 100' returned a non-zero code: 8

Trying out the url http://download.oracle.com/otn-pub/java/jdk/8u73-b02/jdk-8u73-linux-x64.tar.gz shows that the location no longer exist.

I managed to get josso-playground up'n running by cloning the repo, correcting the url in the "oracle-java8/Dockerfile" and then commenting out the following three lines in Vagrantfile:

  #play_cmd << "git clone https://github.com/atricore/atricore-josso-playground.git; "
  #play_cmd << "git -C atricore-josso-playground/oracle-java8 pull origin master; "
  #play_cmd << "docker build -t atricore/josso:oracle-java8 atricore-josso-playground/oracle-java8; "

Then you can start up the box by running "vagrant up --provider virtualbox" from the directory.

Now, in order to fix this I would propose not just to change the url in "oracle-java8/Dockerfile" (thus relying on Oracle), but instead to rely on apt-get install for installing Java.

Question is also what the oracle-java8 docker container is used for... haven't been able to see why its needed, so another solution could be to just remove it completely.

Br.
Jakob Maaloe

**Comment by gbrigandi** [comment]
Hi Jakob,
Thanks for reporting this issue. I'll have a look at that and follow up as soon as it's fixed.
Best,
Gianluca.

**Comment by gbrigandi** [comment]
The oracle-java8 image is the base image of the JOSSO CE server one.  This is due to that JOSSO CE didn't play nice with early releases of OpenJDK, whereas it works fine now, so per your suggestion, I've removed this dependency.  By the way, this image was pulled and built "on the fly" due to Oracle licensing restrictions with Java.
Bear in mind that if you comment out the pull instruction for the playground project, no example configuration will be available for import. So, I'd strongly suggest you keep this unchanged.  

Bottom line, the following changes/fixes have been applied : 

a) Removed completely Oracle's Java dependency
b) Created Vagrant box and Docker images for latest JOSSO CE 2.4.3 and demo environment
c) Ported documentation to the latest JOSSO CE 2.4.3
d) Pushed revised vagrant box and docker images.
  
This kind of issues break the OOTB experience which is a showstopper for anyone kicking tires.
Thanks so much again for bringing this up ! 👍 

Best,
Gianluca.
